operator-framework/pom.xml

```diff
@@ -58,12 +58,6 @@
       <groupId>com.google.testing.compile</groupId>
       <artifactId>compile-testing</artifactId>
       <scope>test</scope>
-      <exclusions>
```

Comment:
Why was this exclusion removed?

Reply:
it was included in jenvtest before, but is not anymore.